db/migrate/20170614135258_create_split_pdf_logs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateSplitPdfLogs < ActiveRecord::Migration
+  def change
+    create_table :split_pdf_logs do |t|
+      t.string :split_pdf_console_log
+      t.string :error_description
+      t.string :uploaded_filetype, null: false
+      t.string :uploaded_filename, null: false
```

Comment:
This column, and others, have names that are too long. In particular, the adjective `uploaded` is redundant.